packages/components/src/List/Toolbar/ColumnChooser/Content/ColumnChooserContent.component.test.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ColumnChooserContent from './ColumnChooserContent.component';
+import { useColumnChooserManager } from '../hooks';
+
+jest.mock('../hooks');
+useColumnChooserManager.mockImplementation(() => ({
+	onChangeVisibility: jest.fn(),
+	onBlurInputTextOrder: jest.fn(),
+	onKeyPressInputTextOrder: jest.fn(),
+	onSelectAll: jest.fn(),
+	onSubmitColumnChooser: jest.fn(),
+	stateColumnChooser: { editedColumns: [], selectAll: false },
+}));
+
+const t = jest.fn((_, translationValue) => translationValue.defaultValue);
+
+describe('ColumnChooserContent', () => {
```

Comment:
Missing actions. Get the actions passed to a column displayer, and call them to check if the closure is correct.